### PR TITLE
Switch to console mode from bash for code block

### PR DESCRIPTION
**Pull request by apc999**
This patch is a proposal to switch the code block in markdown from `bash` language, like
<pre>
```bash
./bin/alluxio fs cat /foo
content
```
</pre>
to `console` language
<pre>
```console
$ ./bin/alluxio fs cat /foo
content1
content2
```
</pre>

to have

```console
$ ./bin/alluxio fs cat /foo
content1
content2
```

Benefits

Current issue:
- Though Github makes the correct randering 

Quick Start Guide - Alluxio master Documentation

### CODE REVIEW

**Comment by ZacBlanco** [comment]
@apc999 is this everything, or is there still more to be done?

**Comment by apc999** [comment]
@ZacBlanco  this is WIP. I will tag you when I am done. thanks!

**Comment by apc999** [comment]
@ZacBlanco PTAL

File: docs/en/compute/Hadoop-MapReduce.md [hunk 1]
```diff
@@ -158,11 +162,11 @@ Hadoop MapReduce users can add `"-Dproperty=value"` after the `hadoop jar` or `y
...
-```bash
-./bin/hadoop jar libexec/share/hadoop/mapreduce/hadoop-mapreduce-examples-2.7.3.jar wordcount \
--Dalluxio.user.file.writetype.default=CACHE_THROUGH \
--libjars {{site.ALLUXIO_CLIENT_JAR_PATH}} \
-<INPUT FILES> <OUTPUT DIRECTORY>
+```console
+$ ./bin/hadoop jar libexec/share/hadoop/mapreduce/hadoop-mapreduce-examples-2.7.3.jar wordcount \
+  -Dalluxio.user.file.writetype.default=CACHE_THROUGH \
+  -libjars {{site.ALLUXIO_CLIENT_JAR_PATH}} \
+  <INPUT FILES> <OUTPUT DIRECTORY>
```
Comment (ZacBlanco) [inline docs/en/compute/Hadoop-MapReduce.md]: Should we make indenting newlines in these console blocks standard? Most of the time we don't do this, but I like when they are indented

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 2]
```diff
@@ -29,51 +29,51 @@ UI in your browser.
...
-           --net=alluxio_nw \
-           --name=alluxio-master \
-           -v ufs:/opt/alluxio/underFSStorage \
-           alluxio/alluxio master
+$ docker run -d \
+  -p 19999:19999 \
+  --net=alluxio_nw \
+  --name=alluxio-master \
+  -v ufs:/opt/alluxio/underFSStorage \
+  alluxio/alluxio master
```
Comment (ZacBlanco) [inline docs/en/deploy/Running-Alluxio-On-Docker.md]: why don't you want to indent this?

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 3]
```diff
@@ -29,51 +29,51 @@ UI in your browser.
...
-           -v ufs:/opt/alluxio/underFSStorage \
-           -e ALLUXIO_JAVA_OPTS="-Dalluxio.worker.memory.size=1G -Dalluxio.master.hostname=alluxio-master" \
-           alluxio/alluxio worker
+$ docker run -d \
+  --net=alluxio_nw \
+  --name=alluxio-worker \
+  --shm-size=1G \
+  -v ufs:/opt/alluxio/underFSStorage \
+  -e ALLUXIO_JAVA_OPTS="-Dalluxio.worker.memory.size=1G -Dalluxio.master.hostname=alluxio-master" \
+  alluxio/alluxio worker
```
Comment (ZacBlanco) [inline docs/en/deploy/Running-Alluxio-On-Docker.md]: same as above comment

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 4]
```diff
@@ -143,20 +143,20 @@ Alluxio uses internal leader election by default.
...
 
-```bash
+```console
 $ docker run -d \
-             ...
-             -e ALLUXIO_JAVA_OPTS="-Dalluxio.master.embedded.journal.addresses=master-hostname-1:19200,master-hostname-2:19200,master-hostname-3:19200 -Dalluxio.master.hostname=master-hostname-1" \
-             alluxio master
+  ...
+  -e ALLUXIO_JAVA_OPTS="-Dalluxio.master.embedded.journal.addresses=master-hostname-1:19200,master-hostname-2:19200,master-hostname-3:19200 -Dalluxio.master.hostname=master-hostname-1" \
+  alluxio master
```
Comment (ZacBlanco) [inline docs/en/deploy/Running-Alluxio-On-Docker.md]: indent?

File: docs/en/deploy/Running-Alluxio-On-Mesos.md [hunk 5]
```diff
@@ -62,8 +62,8 @@ documentation for instructions on how to set configuration properties.
 3. Set the configuration property `alluxio.home` to the path where Alluxio is installed on the Mesos nodes
 4. Launch the Alluxio Mesos framework
 
-```bash
-./integration/mesos/bin/alluxio-mesos-start.sh mesosMaster:5050 // address of Mesos master
+```console
+$ /integration/mesos/bin/alluxio-mesos-start.sh mesosMaster:5050 // address of Mesos master
```
Comment (ZacBlanco) [inline docs/en/deploy/Running-Alluxio-On-Mesos.md]: shouldn't start with a `/`

File: docs/en/operation/2.x-Upgrade.md [hunk 6]
```diff
@@ -30,15 +30,15 @@ journal.
...
+$ bin/alluxio format
 ```
 
 Then start the cluster with the `-i ${BACKUP_PATH}` argument, replacing
 `${BACKUP_PATH}` with your specific backup path.
 
-```bash
-bin/alluxio-start.sh -i ${BACKUP_PATH} all
+```console
+$ bin/alluxio-start.sh -i ${BACKUP_PATH} all
```
Comment (ZacBlanco) [inline docs/en/operation/2.x-Upgrade.md]: `./bin` for consistency?

File: docs/en/operation/2.x-Upgrade.md [hunk 7]
```diff
@@ -30,15 +30,15 @@ journal.
 
 Next, download and untar alluxio-2.x. First format the cluster with
 
-```bash
-bin/alluxio format
+```console
+$ bin/alluxio format
```
Comment (ZacBlanco) [inline docs/en/operation/2.x-Upgrade.md]: `./bin` for consistency?

File: docs/en/operation/2.x-Upgrade.md [hunk 8]
```diff
@@ -13,13 +13,13 @@ This document goes over how to upgrade from Alluxio 1.x to Alluxio 2.x
 alluxio-1.8.1 introduced a journal backup feature. With a 1.8.1 master running,
 create a journal backup by running
 
-```bash
-bin/alluxio fsadmin backup
+```console
+$ bin/alluxio fsadmin backup
```
Comment (ZacBlanco) [inline docs/en/operation/2.x-Upgrade.md]: `./bin/` for consistency?

File: docs/en/ufs/OSS.md [hunk 9]
```diff
@@ -79,8 +79,8 @@ and directories created by Alluxio exist. For this test, you should see files na
 
 Stop Alluxio by running:
 
-```bash
-bin/alluxio-stop.sh local
+```console
+$ bin/alluxio-stop.sh local
```
Comment (ZacBlanco) [inline docs/en/ufs/OSS.md]: `./bin/`?

File: docs/en/ufs/OSS.md [hunk 10]
```diff
@@ -59,18 +59,18 @@ values like `oss-us-west-1.aliyuncs.com` and `oss-cn-shanghai.aliyuncs.com`. Ava
...
 
 This will start an Alluxio master and an Alluxio worker. You can see the master UI at
 [http://localhost:19999](http://localhost:19999).
 
 Run a simple example program:
 
-```bash
-bin/alluxio runTests
+```console
+$ bin/alluxio runTests
```
Comment (ZacBlanco) [inline docs/en/ufs/OSS.md]: `./bin/`

File: docs/en/ufs/OSS.md [hunk 11]
```diff
@@ -59,18 +59,18 @@ values like `oss-us-west-1.aliyuncs.com` and `oss-cn-shanghai.aliyuncs.com`. Ava
 
 Start the Alluxio servers:
 
-```bash
-bin/alluxio format
-bin/alluxio-start.sh local
+```console
+$ bin/alluxio format
+$ bin/alluxio-start.sh local
```
Comment (ZacBlanco) [inline docs/en/ufs/OSS.md]: `./bin` for both?

**Comment by ZacBlanco** [comment]
Do you want to make the CSS changes for removing the ability to select `$` characters in this PR?

File: docs/cn/basic/Command-Line-Interface.md [hunk 12]
```diff
@@ -25,22 +25,22 @@ Usage: alluxio fs [generic options]
 >
 >大多数需要路径参数的命令可以使用通配符以便简化使用，例如：
 >
->```bash
->./bin/alluxio fs rm '/data/2014*'
+>```console
+>$ ./bin/alluxio fs rm '/data/2014*'
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/cn/basic/Command-Line-Interface.md]: they are supposed to be quote. we can decide whether to keep it in a separate PR

File: docs/cn/contributor/Logging-Conventions.md [hunk 13]
```diff
@@ -83,13 +83,13 @@ Alluxio shell附带了一个`logLevel`命令，可以在特定实例上获取或
...
-alluxio logLevel --loggerName = alluxio.heartbeat.HeartbeatContext --target = master，192.168.100.100：30000 --level = DEBUG
+```console
+$ alluxio logLevel --loggerName = alluxio.heartbeat.HeartbeatContext --target = master，192.168.100 .100：30000 --level = DEBUG
 ```
 
 以下命令获取`alluxio.heartbeat.HeartbeatContext`类的所有worker的日志级别
-```bash
-alluxio logLevel --loggerName = alluxio.heartbeat.HeartbeatContext --target = workers
+```console
+$ alluxio logLevel --loggerName = alluxio.heartbeat.HeartbeatContext --target = workers
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/cn/contributor/Logging-Conventions.md]: removed

File: docs/cn/deploy/Running-Alluxio-Fault-Tolerant-on-EC2.md [hunk 14]
```diff
@@ -157,27 +157,27 @@ vagrant ssh AlluxioMaster
...
+$ kill -9 <leader pid found via the above command>
 ```
 
 接着，为了找到新的leader，通过ssh登陆到`AlluxioMaster`节点，该节点上运行着
 [zookeeper](http://zookeeper.apache.org/)，然后运行zookeeper client：
 
-```bash
-/zookeeper/bin/zkCli.sh
+```console
+$ /zookeeper/bin/zkCli.sh
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/cn/deploy/Running-Alluxio-Fault-Tolerant-on-EC2.md]: done

File: docs/cn/deploy/Running-Alluxio-Fault-Tolerant-on-EC2.md [hunk 15]
```diff
@@ -157,27 +157,27 @@ vagrant ssh AlluxioMaster
...
 ```
 
 在zookeeper client终端中，运行以下命令查看当前leader：
 
-```bash
-ls /leader
+```console
+$ ls /leader
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/cn/deploy/Running-Alluxio-Fault-Tolerant-on-EC2.md]: done

File: docs/cn/ufs/MapR-FS.md [hunk 16]
```diff
@@ -63,31 +63,31 @@ alluxio.master.mount.table.root.ufs=maprfs:///<path in MapR-FS>/
 
 也可以将MapR-FS中的某个目录挂载到Alluxio命名空间中：
 
-```bash
-${ALLUXIO_HOME}/bin/alluxio fs mount /<path in Alluxio>/ maprfs:///<path in MapR-FS>/
+```console
+$ bin/alluxio fs mount /<path in Alluxio>/ maprfs:///<path in MapR-FS>/
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/cn/ufs/MapR-FS.md]: done

File: docs/en/advanced/Namespace-Management.md [hunk 17]
```diff
@@ -300,13 +301,13 @@ You can control the active sync interval by changing the `alluxio.master.ufs.act
 
 To disable active sync on a directory, issue the following Alluxio command.
 
-```bash
+```console
 $ bin/alluxio fs stopSync /syncdir
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/advanced/Namespace-Management.md]: done

File: docs/en/advanced/Namespace-Management.md [hunk 18]
```diff
@@ -291,7 +292,7 @@ The feature, called active sync, listens for HDFS events and periodically synchr
 
 To enable active sync on a directory, issue the following Alluxio command.
 
-```bash
+```console
 $ bin/alluxio fs mkdir /syncdir
 $ bin/alluxio fs startSync /syncdir
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/advanced/Namespace-Management.md]: done

File: docs/en/api/POSIX-API.md [hunk 19]
```diff
@@ -55,12 +55,12 @@ specified at `alluxio_path` to the local file system on the specified `mount_poi
...
 in the local file system.
 
-```bash
-bin/alluxio fs mkdir /people
-sudo mkdir -p /mnt/people
-sudo chown $(whoami) /mnt/people
-chmod 755 /mnt/people
-integration/fuse/bin/alluxio-fuse mount /mnt/people /people
+```console
+$ bin/alluxio fs mkdir /people
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/api/POSIX-API.md]: done

File: docs/en/basic/Command-Line-Interface.md [hunk 20]
```diff
@@ -157,21 +157,23 @@ otherwise it returns the current logger level.
 For example, the following command sets the logger level of the class `alluxio.heartbeat.HeartbeatContext` to
 `DEBUG` on master as well as a worker at `192.168.100.100:30000`:
 
-```bash
-alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=master,192.168.100.100:30000 --level=DEBUG
+```console
+$ alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext \
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/basic/Command-Line-Interface.md]: done

File: docs/en/basic/Command-Line-Interface.md [hunk 21]
```diff
@@ -157,21 +157,23 @@ otherwise it returns the current logger level.
...
+  --target=master,192.168.100.100:30000 --level=DEBUG
 ```
 
 And the following command returns the log level of the class `alluxio.heartbeat.HeartbeatContext` among all the workers:
-```bash
-alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=workers
+```console
+$ alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext \
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/basic/Command-Line-Interface.md]: done

File: docs/en/basic/Command-Line-Interface.md [hunk 22]
```diff
@@ -295,8 +298,8 @@ or a path without its header, such as `/<path>`, to use the default hostname and
 >Most of the commands which require path components allow wildcard arguments for ease of use. For
 >example:
 >
->```bash
->./bin/alluxio fs rm '/data/2014*'
+>```console
+>$ ./bin/alluxio fs rm '/data/2014*'
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/basic/Command-Line-Interface.md]: will discuss with you separately on this. blockquote

File: docs/en/compute/Hadoop-MapReduce.md (hunk 1 above)
Comment (apc999) [reply docs/en/compute/Hadoop-MapReduce.md]: agree. better for readability. I will change then when I see these

File: docs/en/deploy/Running-Alluxio-On-Docker.md (hunk 2 above)
Comment (apc999) [reply docs/en/deploy/Running-Alluxio-On-Docker.md]: already indented with two spaces

File: docs/en/deploy/Running-Alluxio-On-Docker.md (hunk 3 above)
Comment (apc999) [reply docs/en/deploy/Running-Alluxio-On-Docker.md]: already indented with two spaces

File: docs/en/deploy/Running-Alluxio-On-Docker.md (hunk 4 above)
Comment (apc999) [reply docs/en/deploy/Running-Alluxio-On-Docker.md]: already indented with two spaces

File: docs/en/deploy/Running-Alluxio-On-Mesos.md (hunk 5 above)
Comment (apc999) [reply docs/en/deploy/Running-Alluxio-On-Mesos.md]: fixed

File: docs/en/operation/2.x-Upgrade.md (hunk 6 above)
Comment (apc999) [reply docs/en/operation/2.x-Upgrade.md]: done

File: docs/en/operation/2.x-Upgrade.md (hunk 7 above)
Comment (apc999) [reply docs/en/operation/2.x-Upgrade.md]: done

File: docs/en/operation/2.x-Upgrade.md (hunk 8 above)
Comment (apc999) [reply docs/en/operation/2.x-Upgrade.md]: done

File: docs/en/ufs/OSS.md (hunk 9 above)
Comment (apc999) [reply docs/en/ufs/OSS.md]: done

File: docs/en/ufs/OSS.md (hunk 10 above)
Comment (apc999) [reply docs/en/ufs/OSS.md]: done

File: docs/en/ufs/OSS.md (hunk 11 above)
Comment (apc999) [reply docs/en/ufs/OSS.md]: done

**Comment by apc999** [comment]
> Do you want to make the CSS changes for removing the ability to select `$` characters in this PR?

No,  can you help me fix this CSS after this PR  is merged?

**Comment by apc999** [comment]
PTAL @ZacBlanco

**Comment by apc999** [comment]
@ZacBlanco please check out ca819ce on the types of code block

**Comment by ZacBlanco** [comment]
Due to the use of backticks, the last change won't render markdown properly
```
* Code blocks should be annotated with the associated file or usage type, e.g.:
    * ```` ```java```` for Java source code
    * ```` ```properties```` for a Java property file
    * ```` ```console```` for an interactive session in shell
    * ```` ```bash```` for a shell script
```

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please

**Pull request merged**